package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "remember-me",
+  "version": "1.0.0",
+  "description": "A simple abstraction of \"remember my previous trip\"",
+  "main": "src/index.js",
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover -x src/view/*.js _mocha --",
+    "lint": "make-up src && echo '🍻  All good!'",
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/holidayextras/remember-me.git"
+  },
+  "keywords": [
+    "local-storage",
+    "cookie"
+  ],
+  "author": "Paul Clarke <paul.clarke@holidayextras.com>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/holidayextras/remember-me/issues"
+  },
+  "homepage": "https://github.com/holidayextras/remember-me#readme",
+  "devDependencies": {
+    "chai": "^3.4.1",
+    "istanbul": "^0.3.22",
+    "make-up": "^5.4.0",
+    "mocha": "^2.3.3",
+    "sinon": "^1.17.2"
+  },
+  "dependencies": {
+    "lodash.merge": "^3.3.2",
+    "tracker": "git@github.com:holidayextras/tracker.git"
```

Comment:
I don't think there's a reasonable way to have a public repo depend on a private one. It makes it impossible for "outsiders" to use, or contribute to, the module, and hooking up functional CI to it introduces security holes.

By the looks of things we only need the `tracker` module for its `error` method. Error handlers could be moved into the module and defaulted to a no-op or to a console warning. As long as we provide a way for it to be injected/overridden, we can still log those events to our (or anyone else's) internal data stores / GA accounts / Bugsnag accounts / whatever - it just has to be done by whatever repo this depends on, not directly by this one.